search/lib/api/wfs.js

```diff
@@ -130,37 +125,36 @@ async function getGranules (request, response) {
   logger.debug(`Event: ${JSON.stringify(event)}`);
```

Comment:
done with this?

Reply:
This has been useful in the past for development, but yeah I think it can be removed now.